cli/app.go

```diff
@@ -25,23 +25,25 @@ const (
 	runFlagData   = "data"
 	runFlagStream = "stream"
 
-	apiKeyCreateFlagOrgID  = "org-id"
-	apiKeyCreateFlagName   = "name"
-	apiKeyFlagMachineID    = "machine-id"
-	apiKeyFlagAliasRobotID = "robot-id"
-	apiKeyFlagLocationID   = "location-id"
-
 	loginFlagDisableBrowser = "disable-browser-open"
 	loginFlagKeyID          = "key-id"
 	loginFlagKey            = "key"
 
+	// Flags shared by api-key, module and data subcommands.
+	generalFlagOrgID        = "org-id"
+	generalFlagLocationID   = "location-id"
+	generalFlagMachineID    = "machine-id"
+	generalFlagAliasRobotID = "robot-id"
```

Comment:
What are your thoughts on this pattern, @maximpertsov? Looks like all four of these got used in a couple places. `apiKeyCreateFlagName` and `moduleFlagName` are both equivalent to "name", but they felt too different semantically to combine into a single `general` flag.

Reply:
this looks great!